data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_kvm_sshd.xml

```diff
@@ -53,13 +53,19 @@
       <xen_append>console=hvc0,115200 console=tty loglevel=5</xen_append>
     </global>
   </bootloader>
-  <partitioning t="list">
+  <partitioning config:type="list">
```

Comment:
this change is not needed. `t` is the abbreation of `config:type=`.

Reply:
ok